client/state/initial-state.js

```diff
@@ -87,16 +88,29 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 	return reduxStore;
 }
 
+// This can't be a middleware as those are invoked before the reducer that
```

Comment:
At very worst, I think I'd still rather the subscribe be bound in `client/boot`, either directly there or as part of it's own library. This doesn't necessarily feel appropriately placed in `initial-state.js`.